sdk/dotnet/SingleReplicationConfig.cs

```diff
@@ -115,12 +115,6 @@ public sealed class SingleReplicationConfigArgs : Pulumi.ResourceArgs
         [Input("enabled")]
         public Input<bool>? Enabled { get; set; }
 
-        /// <summary>
```

Comment:
Breaking change.  Should be noted in the CHANGELOG.  Check upstream to verify that they intentionally made this change.